SolARService_Relocalization/SolARService_Relocalization_properties_cuda.xml

```diff
@@ -43,23 +43,24 @@
             <property name="maxTotalKeypoints" type="uint" value="2000"/>
 	</configure>
     <configure component="SolARDescriptorMatcherKNNOpencv">
-            <property name="distanceRatio" type="float" value="0.9"/>
+            <property name="distanceRatio" type="float" value="0.8"/>
+            <property name="type" type="string" value="BruteForce" description="BruteForce or Flann"/>
     </configure>
     <configure component="SolARGeometricMatchesFilterOpencv">
             <property name="confidence" type="float" value="0.99"/>
-            <property name="outlierDistanceRatio" type="float" value="0.015"/>
-            <property name="epilinesDistance" type="float" value="50.0"/>
+            <property name="outlierDistanceRatio" type="float" value="0.01"/>
+            <property name="epilinesDistance" type="float" value="8.0"/>
     </configure>
     <configure component="SolARPoseEstimationSACPnpOpencv">
             <property name="iterationsCount" type="int" value="500"/>
-            <property name="reprojError" type="float" value="3.0"/>
+            <property name="reprojError" type="float" value="8.0"/>
             <property name="confidence" type="float" value="0.96"/>
             <property name="minNbInliers" type="int" value="15"/>
             <property name="method" type="string" value="USAC_MAGSAC"/>
     </configure>
     <configure component="SolARKeyframeRetrieverFBOW">
             <property name="VOCpath" type="string" value="../../../../../data/fbow_voc/popsift_uint8_indoor.fbow"/>
-            <property name="threshold" type="float" value="0.005"/>
+            <property name="threshold" type="float" value="0.005" description="L2: 0.005; L1:0.5; ChiSquare:0.5; BHATTACHARYYA:0.5; DotProduct:0.01; KLS:500"/>
```

Comment:
Probably not important, but is the description field intended for such kind of comments? Usually it looks more like a documentation of the property itself (what is means, what kind of values are accepted, ...)  than this (maybe I'm wrong).
If you agree, maybe keep it as an XML comment?